specs/backlog.md

```diff
@@ -0,0 +1,92 @@
+- We have the flexibility to move items from a common backlog to sprints without reworking the item.
+- Movement of items through workflow (sprint or otherwise) is reflected in the backlog - eg. an item moved to a "done" state in a sprint should show as "done" in the backlog.
+
+## Problem Description
+
+Several changes are occurring to the MOC teams development models at the same time.
+
+- We are moving to [repository based operations](https://github.com/CCI-MOC/ops-docs/blob/master/specs/devops-mvp.md)
+- Development resources from partners, open source projects, interested operators, and students will play a larger role in the future.
+
+Given the core teams small size we need to change our methodology to enable:
+
+- Knowledge sharing and, more importantly, solution sharing, between team members
+- Capturing those solutions and knowledge, as well as the information identified in section 1 above, in a way that a larger community may find and use it. 
+- The swarming of problems by multiple developers.
+
+A common backlog of MOC issues is one way to capture common understandings and enable rapid on-boarding of development resources. 
+
+The team has settled on GitHub Projects and Issues as the tool of choice. It meets the requirements outlined in the first section above and is tightly coupled with our repository of choice, GitHub. 
+
+## Policy
+
+The items in a backlog should be mutable both in their contents and their relative prioritization. Those changes should occur because of actions of the team, or team members, not due to side effects of the underlying tool.
+
+To ensure that level of order and flexibility issues should be written as close to the code they relate to as possible and also tracked in the backlog for the development team.  
+
+Github supports this model inherently in their [Kanban model](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/about-project-boards) and, with a little bit of work can support it for scrum as well.  <--- we will need to validate by usage that the overhead is not onerous.  Also, a discussion of [Kanban or Scrum](https://www.mountaingoatsoftware.com/blog/when-kanban-is-the-better-choice) --->
+
+1. Issues should be written associated with the repository closest to the section of relevant code.  For example a hil/bmi issues would be written in CCI-MOC/hil. If no repository is appropriate they may be added directly to the [mocbacklog](https://github.com/orgs/CCI-MOC/projects/8).   
+2. If you add the issue to the repository,whenever possible add the repository to [Linked Repositories](https://github.com/orgs/CCI-MOC/projects/8/settings/linked_repositories) under the Menu.  This way it will be included in default searches.  
+3. To find and order all cards available across all cci-moc repositories execute this search in the [add card](assets/githubaddcards.png) dialog:
+ 
+    `org:CCI-MOC state:open is:issue is:open repo:*`
```

Comment:
You don't need the backtics here. If you indent code by four spaces, it will be formatted as a code sample. You can also fence the code like this if you prefer that to indentation:

    ```
    code sample goes here
    ```

Reply:
resolvd in text.